refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,758 @@
+- Decentralised design compatible with both the Internet's Client-Server Architecture and Metaverse/Web3 Architecture
+
+### Design Choices
+
+We have made the following choices:
+- **Language Independence**. The Privacy Request Interchange Vocabulary is independent from any programming language, or any format or language for expressing structured data, and can be materialised in different forms such as json, xml, or other. A [json schema](PRIV.schema.json) is provided for convenience.
+
+- **Rich Semantics**. The Privacy Request Interchange Vocabulary includes `terms` - reserved words to describe common types of Privacy Requests, categories of data, categories of data processing and other key notions. This choice is made to facilitate their uniform interpretation by the implementing systems. Their [human-readable titles and descriptions](dictionary) are provided in json format for convenience.
+
+- **Multiple User Identities**. The Privacy Request Interchange Vocabulary  allows for a Data Subject to be identified using more than one user identity. This choice is made to enable the Privacy Request to be easily exchanged across Systems that use different user identifiers.
+
+- **Systems resolve Privacy Requests**. The Privacy Requests are interpreted at the level of a particular System. If an Organisation operates several Systems, and if the Data Subject wants to have the Privacy Request transmitted to all of them, each System may respond differently. While a Privacy Request can target a group of Systems, the most atomic target of a Privacy Request is thus the System exposing an API for Privacy Requests, and it is only at this level that a Privacy Request can be resolved.
+
+- **Decentralised IDs**. The Privacy Request Interchange Vocabulary uses decentralised ways to uniquely identify Data Subjects, Systems, Requests and their elements. The exchange of Privacy Requests can happen without a centralised entity to control identity disambiguation.
+
+## Proposal
+
+The Privacy Request Interchange Vocabulary includes the following:
+- Key Concepts: [Consent](#consent), [Data Capture](#data-capture), [Data Capture Fragment](#data-capture-fragments), [Demand](#demands), [Privacy Request](#privacy-request), [Privacy Scope](#privacy-scope)
+- Properties: **TBD** full list of properties
+- Terms: **TBD** full list of terms
+
+### Privacy Request
+
+Data Subject is the author of a Privacy Request.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+
+A Privacy Request is made by one and only one Data Subject.
+
+A System MAY have multiple ways to identify the Data Subject, especially when data about them came from some other System that uses different identifiers.
```

Comment:
> The identifiers used to refer to Data Subjects MUST be globally unique

This is specified later in the doc, meaning other systems can't use different identifiers. Am I missing something?

Reply:
@m4rk055 so Systems MAY use different identifiers.

Let us imagine two Systems A and B interoperate, and exchange data about Bob. System A identifies bob with an ID (ID-a), System A sends Bobs data to System B. Systems B uses a different ID (ID-b).

If Systems A and B want to interoperate, and be able to interpret Bobs future Privacy Requests, then:
- System B MUST (in addition to ID-b, also save ID-a)
- System that captures Bobs data MUST obtain consent for `processing-category`:`MATCHING`, `target`:`PARTNERS`
- When Bob makes Privacy Request at System B but wants the Privacy Request transmitted to System A, System B musti include both DSIDs (ID-a & ID-b) in the Privacy Request sent to System A
- If Bob makes a `REVOKE-CONSENT` `processing-category`:`MATCHING`, then System B MUST delete the correspondace between ID-a and ID-b, and there is no more interoperability of future Privacy Requests.

If System A and Systems B use the same ID for Bob, then all this is much simpler.